pandas/core/generic.py

```diff
@@ -5138,9 +5138,13 @@ def describe_1d(data):
         if self.ndim == 1:
             return describe_1d(self)
         elif (include is None) and (exclude is None):
-            if len(self._get_numeric_data()._info_axis) > 0:
+            ncols_numeric = len(self._get_numeric_data()._info_axis)
+            ncols_bool = len(self._get_bool_data()._info_axis)
+            if ncols_numeric > ncols_bool:
```

Comment:
But is is correct! `BoolBlock` inherits from `NumericBlock`, so `ncols_numeric` is always greater or equal than `ncols_bool`. If they differ, we do have non-boolean numeric columns, which we select with `np.number`. If they coincide and `ncols_bool > 0`, the only numeric columns are boolean. The remaining case has no numeric columns &mdash; not even boolean.